jax/numpy/__init__.py

```diff
@@ -432,7 +432,6 @@ def _init():
   # Builds a set of all unimplemented NumPy functions.
   for name, func in util.get_module_functions(np).items():
     if name not in globals() and not name.startswith('_'):
-      _NOT_IMPLEMENTED.append(name)
       globals()[name] = lax_numpy._not_implemented(func, module='numpy')
```

Comment:
Perhaps we should remove this as well? I think having unimplemented functions in the namespace may have been helpful early on, but the remaining functions now are mostly ones we never plan to implement.

Reply:
Done.